src/sentry/static/sentry/app/components/events/interfaces/keyValueList/keyValueListV2.tsx

```diff
@@ -60,6 +64,8 @@ const KeyValueList = ({
                     >
                       {subjectIcon}
                     </ContextData>
+                  ) : valueIsReactRenderable ? (
```

Comment:
I'm not a big fan of nested ternaries ... sometimes it can be very confusing .. wouldn't it be better to create a separate method for rendering the content with early returns?

Reply:
@priscilawebdev I agree. I added you as a reviewer since I know you were the original author of this file. I'll see if I can re-organize this in a way that reads better.